lib/module.js

```diff
@@ -123,6 +123,10 @@ function readPackage(requestPath) {
     return entry;
 
   const jsonPath = path.resolve(requestPath, 'package.json');
+  const rc = stat(jsonPath);
```

Comment:
Don't introduce extra system calls that are practically never necessary.  Handle it inside `InternalModuleReadJSON()`, it's the `CHECK_GE(numchars, 0)` in that function.

Reply:
resolved